### PR TITLE
Various updates to package.json

### PR DESCRIPTION
**Pull request by kethinov**
- Copyedited description (I also applied this to the repo and org description [I also updated the org icon]).
- Updated maintainers/contributors list.
- Bugs URL updated.

### CODE REVIEW

**Comment by kethinov** [comment]
I left the author field with his name to reflect that.

**Comment by brodybits** [comment]
@kethinov I was also wondering what you would think of dropping the LGPL option like I proposed in #11, as a breaking change for a new 0.x release. Should we raise this in an issue first?

**Comment by kethinov** [comment]
I'm for it. No strong opinions on the differences between the open source licenses.

File: package.json [hunk 1]
```diff
@@ -50,11 +55,15 @@
...
+      "email": "kethinov@gmail.com",
+      "web": "https://github.com/kethinov"
     }
   ],
   "bugs": {
-    "email": "jindw@xidea.org",
-    "url": "http://github.com/xmldom/xmldom/issues"
+    "url": "https://github.com/xmldom/xmldom/issues"
   },
   "license": "(LGPL-2.0 or MIT)"
```
Comment (timbru31) [inline package.json]: can you fix the license field, too? To stop npm bugging about it, it needs to be:
```suggestion
   "license": "(LGPL-2.0 OR MIT)"
```

(capitalized OR)

Comment (kethinov) [reply package.json]: Depends on how we want to handle this https://github.com/xmldom/xmldom/pull/11

Comment (brodybits) [reply package.json]: I would vote to close PR #11 and commit the suggestion by @timbru31.

**Comment by kethinov** [comment]
Made that change based on consensus to close https://github.com/xmldom/xmldom/pull/11 in favor of @timbru31's commit.

File: package.json [hunk 2]
```diff
@@ -50,11 +55,15 @@
...
+      "web": "https://github.com/kethinov"
     }
   ],
   "bugs": {
-    "email": "jindw@xidea.org",
-    "url": "http://github.com/xmldom/xmldom/issues"
+    "url": "https://github.com/xmldom/xmldom/issues"
   },
-  "license": "(LGPL-2.0 or MIT)"
+   "license": "(LGPL-2.0 OR MIT)"
```
Comment (brodybits) [inline package.json]:
```suggestion
  "license": "(LGPL-2.0 OR MIT)"
```

remove extra space

**Comment by kethinov** [comment]
Seems good now.

**Pull request merged**